trace-sdk/src/main/java/io/bitrise/trace/utils/log/LogMessageConstants.java

```diff
@@ -50,6 +50,7 @@
     public static final String SET_URL_STREAM_HANDLER_FACTORY_FAILED = "setURLStreamHandlerFactory failed. This means" +
             " the Trace SDK will not be able to capture network metrics that use URLConnection. Please refer to the " +
             "documentation for further guidance.";
+    public static final String TRACE_DEBUG_FLAG_STATUS = "TraceSdk debug = %b";
```

Comment:
nit: I would make this message more verbose:  if I only see this in the logs I may not now what would it mean.

Reply:
Yeah I agree - but its really hard 😅 

Have updated it to read `TraceSdk debug mode enabled = %b` - does that sound any better? 

Would it be better if it said it was in production/debug mode ? 

I think we should also be updating the readme here to include what it means - i'll add that into this PR.

Reply:
I think this simple modification in the wording helps a lot in the understanding, thanks!